Comments/components/commentBrowser.js

```diff
@@ -42,12 +45,22 @@ export class CommentBrowser extends Window {
       this.initializeContainers();
       this.initializeExtents();
       this.relayoutWindow();
-      $world.addMorph(this);
+
       instance = this;
+      this.name = 'comment browser';
+      this.commentGroups = {}; // dict Morph id -> Comment group morph
+
+      this.makeVisible();
```

Comment:
`this.makeVisible();`
Könnte diese Zeile nicht außerhalb der Verzweigung stehen und nach ihr aufgerufen werden?